app/vmstorage/main.go

```diff
@@ -401,7 +408,11 @@ var (
 	staleSnapshotsRemoverWG sync.WaitGroup
 )
 
-var activeForceMerges = metrics.NewCounter("vm_active_force_merges")
+var (
+	activeForceMerges     = metrics.NewCounter("vm_active_force_merges")
+	snapshotsFailedTotal  = metrics.NewCounter("vm_snapshots_failed_total")
+	snapshotsCreatedTotal = metrics.NewCounter("vm_snapshots_created_total")
```

Comment:
wdyt about using `vm_http_requests_total` metric instead?
It used for listing other /admin actions like `delete/{}/prometheus/api/v1/admin/tsdb/delete_series` and also has `vmagent_http_request_errors_total` counterpart.
I was kinda expecting to find `/snapshot` label in these metrics but I didn't)
You can also cover /list and /delete path parts if you go this way.